src/Sylius/Bundle/ApiBundle/Event/ResetPasswordRequested.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Event;
+
+class ResetPasswordRequested
```

Comment:
```suggestion
final class ResetPasswordRequested
```

Reply:
Events can be left without final as well